### PR TITLE
Update README for folks like me

### PR DESCRIPTION
**Pull request by ryanleesipes**
I had a hard time figuring out exactly how to build the website, so I've added documentation to help viewers like me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ git clone https://github.com/mozilla/product-details-json
 sudo yum install npm
 sudo npm install -g less 
 ```
+#### Note: The specific version of less used and known to work is 2.7.2, it can be installed via the following command:
+```
+sudo npm install -g less@2.7.2`
```
Comment (Sancus) [inline README.md]: I think you should just replace the generic install command, and change the note to something like "Less versions above 3.0 won't work." It's confusing to have to go back and change your less version if you accidentally run the generic command first on a distro that has Less 3 by default like Ubuntu 17.

Comment (ryanleesipes) [reply README.md]: Yeah, I'll take care of that now. Were there big change in 3.0? Can we fix that so that it does work?

**Comment by ryanleesipes** [comment]
@Sancus updated, let me know what you think.

**Comment by Sancus** [comment]
This looks good. You should squash your adjusted commit into the original one so there's only one commit, though.

Also I'd prefer you changed the commit message to something specific, "folks like me" is pretty vague. " Better would be: "Update README with LESS 2.7.2 and python simple server instructions."

I'm not sure if you know how to squish commits, so:

`git reset --soft HEAD~2`
`git commit -m "Update README with LESS 2.7.2 and python simple server instructions."`

After that you'll need to force push:
`git push -f origin patch-1`

**Comment by kewisch** [comment]
Note you can also have github do it by clicking the merge button dropdown and selecting the squash option. We can even disallow merge commits, which we should consider doing

**Comment by Sancus** [comment]
Oh huh, that option was right in front of me and I missed it. Been a year or two since I've used github cooperatively I guess. Thanks!

**Comment by kewisch** [comment]
Another protip, the “allow edits by maintainers” option is on by default, which would allow you to push to Ryan’s PR branch and make changes

**Pull request merged**